sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/environment/KeyVaultOperation.java

```diff
@@ -205,21 +203,4 @@ void setProperties(HashMap<String, String> properties) {
         this.properties = properties;
     }
 
-    boolean isUp() {
```

Comment:
In main branch, it's used for KeyVaultHealthIndicator. Why KeyVaultHealthIndicator is deleted in 4.0 branch?

Reply:
we use another method for kv health indicator now, and we do not have the kv health indicator for property source now.

Reply:
I see.
Now for key vault secrets, we only have `KeyVaultSecretHealthIndicator` which only check health of `SecretAsyncClient`.

We can add health indicator for property source if customer ask for this feature in the future.

Reply:
Could you please add this into changelog?

Reply:
will do